data-collection/modules/physiological/heart-rate.md

```diff
@@ -0,0 +1,29 @@
+---
+sidebar_position: 6
+title: Heart Rate
+---
+
+** Tracking the track heart to identify irregularities **
+
+The Heart Rate module is used to record how many times a Patient’s heart beats in a minute. The normal value is between 60 and 100 bpm, but can be affected by certain heart conditions, stress, anxiety, etc.
+
+Recording this information is an easy and effective way for a Care Team to gauge a Patient’s health.
+
+## How it works
+
+Patients can measure their Heart Rate using a fitness tracker, smartphone with relevant functionality, smart watch, or counting the pulse in the wrist in one minute. From there, Clinicians can view new and historic results in the Clinician Portal and be informed on severity.  
```

Comment:
Patients can collect their Heart Rate by any of the 3 sources below: 
- connecting the Huma to their Apple Health or Google Fit app
- manually inputing 
- resting their index finger over their smartphone cameras
The smartphone camera Heart Rate collection also gives a Heart Rate Variability and a Classification reading. The Administrator can configure which readings and sources (maximum 2 sources per deployment) the deployment needs for their delivery of care or research. 

From there, Clinicians can view new and historic results in the Clinician Portal and be informed on severity.